source/_integrations/amcrest.markdown

```diff
@@ -92,7 +92,7 @@ authentication:
   required: false
   type: string
   default: basic
-scan_interval:
+scan_interval: (Obsolete from 108.5)
```

Comment:
This breaks the build

Reply:
Is there a deprecated format for this? Sorry @frenck, i have no idea of the process to change and resumit this within github.

Would this work?

```
scan_interval

    (integer)(Optional)(Before Version 108.5)

    Defines the update interval of the sensor in seconds.

    Default value:

    10

```

Reply:
Just put it in the description?